src/Core/Domain/Product/Pack/Command/SetPackProductsCommand.php

```diff
@@ -90,11 +90,10 @@ private function setProducts(array $products): void
 
         foreach ($products as $product) {
             $this->assertQuantity($product['quantity']);
-
             $this->products[] = new QuantifiedProduct(
-                $product['product_id'],
-                $product['quantity'],
-                isset($product['combination_id']) ? $product['combination_id'] : null
+                (int) $product['product_id'],
+                (int) $product['quantity'],
+                (int) ($product['combination_id'] ?? null)
```

Comment:
Ok for the first two, but not this one You change the behavior here by injecting `0` instead of `null` (since you cast the null) That's probably why we didn't use the `??` here initially

Reply:
I revert. I was not confortable with this replace ... I know now why.